SamplesDashboard/SamplesDashboard/Controllers/SamplesController.cs

```diff
@@ -51,15 +51,15 @@ public async Task<IActionResult> GetSamplesListAsync()
         [Route("api/[controller]/{id}")]
         public async Task<IActionResult> GetDependenciesAsync(string id)
```

Comment:
>GetDependenciesAsync [](start = 41, length = 20)

the naming here now starts to introduce a little confusion since we're not returning Dependency objects but rather Repository.  It feels like this should be renamed as well as the one in SampleService.